[
    {
        "content": "<p>I stuck in this error message when trying to login to dataverse through OIDC:</p>\n<p>[2024-03-14T08:13:01.209+0000] [Payara 6.2023.8] [WARNING] [] [jakarta.enterprise.web] [tid: _ThreadID=69 _ThreadName=http-thread-pool::http-listener-1(1)] [timeMillis: 1710403981209] [levelValue: 900] [[<br>\n  StandardWrapperValve[Faces Servlet]: Servlet.service() for servlet Faces Servlet threw exception<br>\njava.lang.IllegalArgumentException: URI is not absolute<br>\n    at java.base/java.net.URL.fromURI(URL.java:721)<br>\n    at java.base/java.net.URI.toURL(URI.java:1139)<br>\n    at com.nimbusds.oauth2.sdk.http.HTTPRequest.toURLWithUncheckedException(HTTPRequest.java:236)<br>\n    at com.nimbusds.oauth2.sdk.http.HTTPRequest.&lt;init&gt;(HTTPRequest.java:230)<br>\n    at com.nimbusds.oauth2.sdk.TokenRequest.toHTTPRequest(TokenRequest.java:514)<br>\n    at edu.harvard.iq.dataverse.authorization.providers.oauth2.oidc.OIDCAuthProvider.getAccessToken(OIDCAuthProvider.java:267)<br>\n    at edu.harvard.iq.dataverse.authorization.providers.oauth2.oidc.OIDCAuthProvider.getUserRecord(OIDCAuthProvider.java:223)<br>\n    at edu.harvard.iq.dataverse.authorization.providers.oauth2.OAuth2LoginBackingBean.exchangeCodeForToken(OAuth2LoginBackingBean.java:103)<br>\n    at edu.harvard.iq.dataverse.authorization.providers.oauth2.OAuth2LoginBackingBean$Proxy$_$$_WeldSubclass.exchangeCodeForToken(Unknown Source)<br>\n    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)</p>",
        "id": 426490282,
        "sender_full_name": "Mohsen Jafari",
        "timestamp": 1710404780
    },
    {
        "content": "<p><span class=\"user-mention silent\" data-user-id=\"698782\">Mohsen Jafari</span> has marked this topic as resolved.</p>",
        "id": 426513966,
        "sender_full_name": "Notification Bot",
        "timestamp": 1710413099
    }
]